$OpenBSD: patch-capplets_about-me_gnome-about-me_glade,v 1.2 2008/11/21 17:02:49 jasper Exp $

XXX:
Changing the password doesn't work like it should at the moment.
Hide the button for now.

--- capplets/about-me/gnome-about-me.glade.orig	Tue Nov 11 20:59:01 2008
+++ capplets/about-me/gnome-about-me.glade	Tue Nov 11 20:59:40 2008
@@ -200,22 +200,6 @@
 		      <property name="fill">False</property>
 		    </packing>
 		  </child>
-
-		  <child>
-		    <widget class="GtkButton" id="password">
-		      <property name="visible">True</property>
-		      <property name="can_focus">True</property>
-		      <property name="label" translatable="yes">Change Passwo_rd...</property>
-		      <property name="use_underline">True</property>
-		      <property name="relief">GTK_RELIEF_NORMAL</property>
-		      <property name="focus_on_click">True</property>
-		    </widget>
-		    <packing>
-		      <property name="padding">0</property>
-		      <property name="expand">False</property>
-		      <property name="fill">False</property>
-		    </packing>
-		  </child>
 		</widget>
 		<packing>
 		  <property name="padding">0</property>
